Assets/_BForBoss/_Character/Scripts/PlayerDashBehaviour.cs

```diff
@@ -29,18 +25,16 @@ public class PlayerDashBehaviour : MonoBehaviour
 
         private ECM2.Characters.Character _baseCharacter = null;
         private InputAction _dashInputAction = null;
-        private Action _onDashing = null;
+        private Action _isDashActivated = null;
```

Comment:
Where is this being used? also, the variable is phrased as if it is supposed to be a bool, suggest a name change.

Reply:
i'll revert it.